javascript/javascript1/week3/travel-tracker.js

```diff
@@ -0,0 +1,14 @@
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 432,
+};
+
+function getTravelTime(object) {
+  let numValue = (object.destinationDistance / object.speed) * 60; // getting number of minutes
```

Comment:
Super! Can you maybe think of a better name for your let ?

Reply:
Will do now :)